au3/libraries/au3-module-manager/PluginDescriptor.cpp

```
@@ -75,6 +75,11 @@ const wxString& PluginDescriptor::GetVendor() const
     return mVendor;
 }
 
+const wxString& PluginDescriptor::GetDescription() const
```

Cppcheck (2.20.0)

[style] The function 'GetDescription' is never used.

(unusedFunction)